src/Hydrators/CatHydrator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace TopCat\Hydrators;
+
+use TopCat\Entities\CatEntity;
```

Comment:
This doesn't appear to be used. Is it necessary?

Reply:
doing